package/boot/uboot-envtools/files/ramips

```diff
@@ -40,6 +40,11 @@ xiaomi,mir3g|\
 xiaomi,redmi-router-ac2100)
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x1000" "0x20000"
 	;;
+hootoo,tm05)
```

Comment:
Alphabetic sorting, please.

Reply:
Sorry, missed that! As some of the items have several entries, I went by the first on the list - but yell if this isn't how you want it.

Thanks!